src/index.js

```diff
@@ -0,0 +1,239 @@
+'use strict';
+
+// 2. Change temperature by clicking on an arrow
+
+const state = {
+  temp: 55,
+};
+let flagFahrenheit = true;
+const today = new Date();
+
+const increaseTemp = () => {
+  state.temp += 1;
+  const tempContainer = document.getElementById('current_temp');
+  tempContainer.textContent = `Current temp: ${state.temp}`;
+  tempContainer.textContent = flagFahrenheit
+    ? `${Math.trunc(state.temp)}°F`
+    : `${Math.trunc(state.temp)}°C`;
+  changeColorTemp(state.temp);
+  changeBackground(state.temp);
+};
+
+const registerEventHandlers = () => {
+  const increaseTempButton = document.getElementById('increaseTempButton');
+  increaseTempButton.addEventListener('click', increaseTemp);
+};
+document.addEventListener('DOMContentLoaded', registerEventHandlers);
+
+const decreaseTemp = () => {
+  state.temp -= 1;
+  const tempContainer = document.getElementById('current_temp');
+  tempContainer.textContent = `Current temp: ${state.temp}`;
+  tempContainer.textContent = flagFahrenheit
+    ? `${Math.trunc(state.temp)}°F`
+    : `${Math.trunc(state.temp)}°C`;
+  changeColorTemp(state.temp);
+  changeBackground(state.temp);
+};
+
+const registerEventHandlersDecrease = () => {
+  const increaseTempButton = document.getElementById('decreaseTempButton');
+  increaseTempButton.addEventListener('click', decreaseTemp);
+};
+document.addEventListener('DOMContentLoaded', registerEventHandlersDecrease);
+
+// Temperature Ranges Change Landscape
+
+const changeBackground = (temp) => {
+  const element = document.getElementById('main__intro__right');
+  if (!flagFahrenheit) {
+    temp = (9 / 5) * temp + 32;
+  }
+  if (temp >= 80) {
+    element.className = 'summer';
+  } else if (temp < 80 && temp >= 54) {
+    element.className = 'spring';
+  } else if (temp < 54 && temp >= 34) {
+    element.className = 'autumn';
+  } else {
+    element.className = 'winter';
+  }
+};
+// Temperature Ranges Change Temperature
+
+const changeColorTemp = (temp) => {
+  const tempContainer = document.getElementById('current_temp');
+  if (!flagFahrenheit) {
+    temp = (9 / 5) * temp + 32;
+  }
+  if (temp < 50) {
+    tempContainer.className = 'teal';
```

Comment:
I know that the demo app did this but class names should reflect meaning, not what the style should look like.

If we changed the style sheet to go from red to purple to blue having the medium temperature class be named `yellow` is confusing.